atom/browser/api/atom_api_web_contents.cc

```diff
@@ -2548,7 +2552,11 @@ v8::Local<v8::Value> WebContents::TabValue() {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
 
   std::unique_ptr<base::DictionaryValue> value(
-    ExtensionTabUtil::CreateTabObject(web_contents())->ToValue().release());
+      ExtensionTabUtil::CreateTabObject(
+          web_contents(), ExtensionTabUtil::kScrubTab,
```

Comment:
This should be `kDontScrubTab` or browser-laptop will not have url, title and favicon_url info when calling `tabValue`

Reply:
Fixed in https://github.com/brave/muon/commit/70b5f858d69560c056ba6688fced19e9a1c62e74